lib/api/wpapi.js

```diff
@@ -0,0 +1,77 @@
+var $ = require('zepto-browserify').$;
+
+var name = 'WP.CORE';
+
+var baseUrl = 'https://public-api.wordpress.com/wp/v2';
```

Comment:
Oh sorry @timmyc I haven't understood at first what you were saying about the namespaces :). Yes, let's add that to the version selector, It's easy enough I think.